### PR TITLE
Fixes IC filter and adds OOC filter

### PR DESCRIPTION
**Pull request by ike709**
Mark forgot a PR.

### CODE REVIEW

File: code/modules/goonchat/browserassets/js/browserOutput.js [hunk 1]
```diff
@@ -576,7 +590,16 @@ function ehjaxCallback(data) {
...
+		} else if (data.syncRegex) {
+			for (var i in data.syncRegex) {
+
+				var regexData = data.syncRegex[i];
+				var regexName = regexData[0];
+				var regexFlags = regexData[1];
+				var regexReplaced = regexData[2];
+
+				replaceRegexes[i] = [new RegExp(regexName, regexFlags), regexReplaced];
+			}
```
Comment (MarkSuckerberg) [inline code/modules/goonchat/browserassets/js/browserOutput.js:602]: I think there needs to be another closing bracket here, but I'm not 100% on that. Just seems to fit the description the error gave

File: code/controllers/configuration/configuration.dm [hunk 2]
```diff
@@ -374,19 +375,39 @@
 	return runnable_modes
 
 /datum/controller/configuration/proc/LoadChatFilter()
-	GLOB.in_character_filter = list()
+	var/list/in_character_filter = list()
+	var/list/ooc_filter = list()
+
+	if(!fexists("[directory]/ooc_filter.txt"))
+		log_config("Error 404: ooc_filter.txt not found!")
```
Comment (qwertyquerty) [inline code/controllers/configuration/configuration.dm:382]: Please tell me this is just for the memes

Comment (ike709) [reply code/controllers/configuration/configuration.dm:382]: It is.

**Comment by ike709** [comment]
Works fine locally.

**Review by qwertyquerty — APPROVED** [review]

**Pull request merged**